storage-access.bs

```diff
@@ -98,6 +98,18 @@ Script in the <{iframe}> can call |doc|`.`{{Document/hasStorageAccess()}} to det
 
 </div>
 
+This specification also defines a method for a [=site=] to request access to its [=unpartitioned data=] when later loaded in a [=third party context=] on another specified [=site=] ({{Document/requestStorageAccessUnderSite()}}), and a method for a [=site=] to allow such requests to succeed ({{Document/allowStorageAccessRequestOnSite()}}).
```

Comment:
I'm not sure if it's correct to say that a [site](https://html.spec.whatwg.org/multipage/origin.html#site) (which is a tuple) requests access to something. Maybe just remove the links?

Reply:
Ah, I was copying language from line 113. I'll just remove these links.